com.unity.ml-agents/Runtime/Policies/HeuristicPolicy.cs

```diff
@@ -29,7 +29,10 @@ public HeuristicPolicy(Func<float[]> heuristic)
         public void RequestDecision(AgentInfo info, List<ISensor> sensors)
         {
             StepSensors(sensors);
-            m_LastDecision = m_Heuristic.Invoke();
+            if (!info.done)
```

Comment:
We clear the `done` flag right be calling RequestDecision - not sure this will have any effect.
https://github.com/Unity-Technologies/ml-agents/blob/6a1107185eaf045d7f7e233edfa8885f833cde9b/com.unity.ml-agents/Runtime/Agent.cs#L623-L627

Reply:
Yes, I want to avoid it being called when using `NotifyAgentDone()`.
It does have an effect in that regard...